workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -242,25 +243,27 @@ groups <- list(
                layout=c('layout')
                )
 
-for (group in groups){
-    mdcat('### ', group)
-    print(plotPCA(rld, intgroup=group))
+# NOTE: Here we make interactive PCA plots using ggplotly. However, this does
+#    not work inside a loop. So, if you have multiple groups, copy and paste
+#    the following lines, updating the variable 'i'.
+p <- list()
+for(group in groups){
+    p[[group]] <- plotPCA.ly(rld, intgroup=group)
 }
-```
-
 
-## Most-varying genes
+i <- 1
```

Comment:
Unfortunate that ggplotly has that limitation, but I think this is the cleanest workaround.